### DISCUSSION TITLE
Adding the IPOK and IFAK from DDA.

### DISCUSSION THREAD

**Discussion by ghost** [cataclysmbn/Cataclysm-BN#2566] [General]
DDA added Individual Patrol Officer Kits (IPOK) and Individual First Aid Kits (IFAK), A police and military version of first aid kits. I would specifically like to add these into BN and replace some first aid kit spawns (possibly other medical spawns too) with these.

The IPOK is a heavy downgrade to the first aid kit, having only some bandages, a tourniquet item we could replace with more bandages, and flavor items. This is a small kit that goes inside a pocket and would be a replacement for first aid kits on police zombies.

The IFAK is also a downgrade, having no disinfectant or painkillers but having quite a few bandages, purification tablets, and hemostatic powder, plus medical tape and some flavor items. we can replace adhesive bandages with regular bandages, as well as medical scissors with regular scissors.

Option 1: we implement as-is, after replacing DDA items with equivalents, both are downgrades that would nerf the loot tables. 

Option 2: we consider adding some form of disinfectant / painkillers to either or both. IPOK could still remain a small item or

Their JSONs were posted here inside the discord. https://discord.com/channels/830879262763909202/830916451517857894/1092911225702191134

The loot drops we would change firstly, would be under monsterdrops, zombie_cop and swat_zombie_gear to IPOK, and under the military item group infantry_medical_gear mil_food_nodrugs to IFAK.

**Comment by ghost** [comment]
This flavor expansion is better suited towards a mod, which I'll make at some point to bridge the missing pieces of military and police flavor between the forks.

**Reply by DrPariah** [reply]
I was trying to figure out how to say that when I saw the Discussion post here. My recommendation would be to use the IFAK/IPOK as an opportunity to distribute Splints and Hemostatic Powder 'in the wild' to diversify the available medical supplies looted outside of stores and the like. Both those items are useful, especially having some Splints that you don't need crafted. For utility I would recommend that they still come with small amounts of disinfectant and painkiller, much smaller than a 'regular' First Aid Kit, but like you said upping the amount of bandages and/or gauze. This makes them immediately useful in the field.
If it's possible, I wouldn't recommend messing with the item spawns for police/military roadblocks. Those, logically, _should_ have full First Aid Kits because they have vehicles there. But that, along with the above, are all just suggestions on my part.
